shell/common/gin_helper/function_template.h

```diff
@@ -225,7 +226,7 @@ class Invoker<IndicesHolder<indices...>, ArgTypes...>
   void DispatchToCallback(base::Callback<void(ArgTypes...)> callback) {
     v8::MicrotasksScope script_scope(args_->isolate(),
                                      v8::MicrotasksScope::kRunMicrotasks);
-    callback.Run(ArgumentHolder<indices, ArgTypes>::value...);
+    callback.Run(std::move(ArgumentHolder<indices, ArgTypes>::value)...);
```

Comment:
Will this be future proof ? Expecting all arguments to be movable ?

Reply:
If the parameter is a const reference, the `ArgTypes&&` would be converted to `const ArgTypes&` automatically.

If the parameter is a value, the `ArgTypes&&` would be converted to `ArgTypes` and C++ would try to invoke the copy constructor.

The only difference is when the parameter is a rvalue reference or when it is a value but has move constructor, and in this case move constructor would be used. So this should be future proof.